pipelines_template.yml

```diff
@@ -34,7 +34,7 @@ pipelines:
       ################################################################################
       prepare_finished:
         script:
-          - ./flow contentReleasePrepare:createContentRelease {{ .contentReleaseId }} {{ .__jobID }}
+          - ./flow contentReleasePrepare:createContentRelease {{ .contentReleaseId }} {{ .__jobID }} {{ .workspaceName }}
```

Comment:
right now, the workspaceName is in lots of APIs, and I am not so sure about that right now. I feel this pollutes lots of APIs and makes the control flow rather difficult to understand.

Right now, it is: "ContentReleaseManager has workspaceName arg -> stored as ".workspaceName" when evaluating the pipelines_template", then added to the content release,  and then taken from there for the individual rendering.

I'd rather propose an alternative, and I hope more simple approach:

- in ContentReleaseManager, we pass in the workspace name via additionalVariables.
- then, in pipelines_template.yml, we can access the workspace name.
- we can directly append the workspace argument to `./flow nodeEnumeration:enumerateAllNodes`.

(this case should be documented in the README and in pipelines_template.yml)

Reply:
My reasoning was that the workspace is something more relevant than `additionalVariables ` and should be validated early on.
I actually liked that in the core API I have it explicitly defined and one has to keep it in mind every time one works with the CR. Especially when further work is done by other contributors on the package.
Also when flipping the merging of the additionalVariables we cannot rely anymore on the `workspaceName` being defined there except if we merge 3 arrays or defined a fallback.
Maybe a short call is better to come to a common conclusion 🙂